common/http/pom.xml

```diff
@@ -27,19 +27,28 @@
     <modelVersion>4.0.0</modelVersion>
     <artifactId>helidon-common-http</artifactId>
     <name>Helidon Common HTTP</name>
+    <description>Common HTTP primitives used by both blocking and reactive servers</description>
 
     <dependencies>
         <dependency>
             <groupId>io.helidon.common</groupId>
-            <artifactId>helidon-common-reactive</artifactId>
+            <artifactId>helidon-common</artifactId>
```

Comment:
![image](https://user-images.githubusercontent.com/1773630/184705677-c6157230-0000-42b3-acc9-1b85d40fa65f.png)

Reply:
Common http is now used by both Helidon SE and new blocking server. Cannot have reactive in. Sorry

Reply:
The end of an era it is